libs/ui/core/src/lib/brain/typeof.pipe.ts

```diff
@@ -0,0 +1,8 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({ name: 'typeofString', standalone: true })
+export class TypeOfStringPipe implements PipeTransform {
+	transform(value: unknown): value is string {
+		return typeof value === 'string';
```

Comment:
We could probably just use a function call in the  template of the component. I am cool with keeping this though if you think we should

Reply:
I'm not a huge fan of function calls in templates, but I guess now that signals are coming and are simply functions I guess it's not a big deal to replace the pipe with one.